app/components/Home.tsx

```diff
@@ -21,8 +23,14 @@ export default function Home(props: {
   // HOOKS
   const { data } = useTina(props);
 
+  const [width, setWidth] = useState(0);
+
+  useEffect(() => {
+    setWidth(typeof window !== "undefined" ? window.innerWidth : 0);
+  }, []);
+
   return (
-    <Grid templateColumns={"1fr"}>
+    <Grid templateColumns={"1fr"} background={"white"}>
```

Comment:
This messes up background images
```suggestion
    <Grid templateColumns={"1fr"}>
```

Reply:
Yep, that was it! I also wrapped the Machines component in a Box with the background set to white, full width.